hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestSchemaEvolution.java

```diff
@@ -184,17 +185,18 @@ private void writeTableWithSchema1(TableOptions tableOptions) throws ExecutionEx
         + "  cast(gender as char),"
         + "  cast(age as int),"
         + "  cast(ts as timestamp),"
+        + "  cast(struct_col as row<f0 int, f1 string>),"
         + "  cast(`partition` as string) "
```

Comment:
`struct_col` -> `f_struct`, whatever, it's just a preference.

Reply:
Done!